client/dashboard/app/primary-menu/index.tsx

```diff
@@ -11,6 +11,9 @@ function PrimaryMenu() {
 				<ResponsiveMenu.Item to="/overview">{ __( 'Overview' ) }</ResponsiveMenu.Item>
 			) }
 			{ supports.sites && <ResponsiveMenu.Item to="/sites">{ __( 'Sites' ) }</ResponsiveMenu.Item> }
+			{ supports.stores && (
+				<ResponsiveMenu.Item to="/sites">{ __( 'Stores' ) }</ResponsiveMenu.Item>
```

Comment:
Sorry if this has been raised elsewhere, but why are we using the "/sites" route for stores? I can understand wanting to have compatibility (e.g. links from Jetpack or other wp-admin views back into the HD when they don't know what garden dashboard they should go to). But if we want route compatibility, shouldn't we also reuse the same router for sites and stores? We shouldn't have a `router/stores.tsx`. Otherwise the route will just diverge.

I think I'm in favour of just going all in on having separate `/sites` and `/stores` routes though.

Reply:
Mentioned in https://github.com/Automattic/wp-calypso/pull/106182#discussion_r2397215647. It seems more complex to replace all `/sites` to `/stores`. If we only replace the path of the sites list, it seems a bit inconsistency.